package.json

```diff
@@ -79,6 +79,6 @@
   },
   "packageManager": "pnpm@8.7.6",
   "engines": {
-    "node": "^14.18.0 || >=16.10.0"
+    "node": ">=16.10.0"
```

Comment:
Is this change strictly required?

Reply:
Nuxi CLI bundle (today) is compatible but it is so likely we will break it unnoticed with any of the bundled deps.

Also nuxt/nitro and even pnpm are incompatible with any command! It just makes sense to let users be aware of this as a strict requirement: (and there is always escape hack with skip-engines flag for rare conditions someone want to make Nuxt 3 compatible with Node 14...)

<img width="1129" alt="image" src="https://github.com/nuxt/cli/assets/5158436/1b6d7d68-b095-47c5-8d75-890393ffb736">